channels/stable-4.10.yaml

```diff
@@ -0,0 +1,6 @@
+feeder:
+  delay: PT0H
+  filter: 4\.[9\|10]\.[0-9].*
```

Comment:
We don't want 4.9.z in `stable-4.10` until our post-GA cooking has completed.  See [this example][1].

[1]: https://github.com/openshift/cincinnati-graph-data/blob/7b5034bff7859f81dd8b5df12777d4fdb2d3258a/channels/stable-4.9.yaml#L3